articles/azure/azs-how-create-vm-image-portal.md

```diff
@@ -0,0 +1,119 @@
+---
+title: How to create a custom image from a VM on Azure Stack - Portal | UKCloud Ltd
```

Comment:
How to create a custom image from a VM on Azure Stack using the UKCloud Azure Stack portal

(for consistency with other articles)